shot-android/src/main/java/com/karumi/shot/ScreenshotTest.kt

```diff
@@ -131,9 +133,26 @@ interface ScreenshotTest {
     private fun disableAnimatedComponents(view: View) {
         runOnUi {
             hideEditTextCursors(view)
+            hideScrollViewBars(view)
         }
     }
 
+    private fun hideScrollViewBars(view: View) {
+        view.childrenViews<ScrollView>().forEach {
+            hideViewBars(it)
+        }
+
+        view.childrenViews<HorizontalScrollView>().forEach {
+            hideViewBars(it)
+        }
+    }
+
+    private fun hideViewBars(it: View) {
+        it.isHorizontalScrollBarEnabled = false
+        it.isVerticalScrollBarEnabled = false
```

Comment:
I've found that adding `it.scrollBarSize = 0` helps to completely disable them

Reply:
If this implementation works, it is ok for me right now.